spring-cloud-netflix-core/src/test/java/org/springframework/cloud/netflix/feign/invalid/FeignClientValidationTests.java

```diff
@@ -42,10 +43,7 @@
 	@Test
 	public void testNameAndValue() {
 		this.expected.expectMessage("only one is permitted");
-		AnnotationConfigApplicationContext context = new AnnotationConfigApplicationContext(
-				NameAndValueConfiguration.class);
-		assertNotNull(context.getBean(NameAndValueConfiguration.Client.class));
```

Comment:
The exceptions happen at construction time of the AnnotationConfigApplicationContext. The assertions after that were dead code.